lib/Models/CustomComponents.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
+import React from 'react';
```

Comment:
My previous comment was a bit academic, but it's definitely no good to actually require `React` inside a module in `Models`.